<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtserialterminal</class>
 <widget class="QMainWindow" name="qtserialterminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtserialterminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial 8bit-stop1-noparity-noflow</string>
    </property>
    <widget class="QComboBox" name="cmbBaud">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>79</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>BaudRate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmbPort">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>79</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ttyUSB0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PortName</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkLocalEcho">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Local Echo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Automatic Request</string>
    </property>
    <widget class="QLineEdit" name="txtReqInterval">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Interval</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDelay">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkUseDelay">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Delay</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtReqData">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Request String</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkNewRow">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&quot;\r&quot;</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkNewLine">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&quot;\n&quot;</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>701</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Terminal</string>
    </property>
    <widget class="QPlainTextEdit" name="txtTerminal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>661</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionClear"/>
   <addaction name="actionAboutQt"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>aboutQt</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
